modules/olm-mirroring-catalog.adoc

```diff
@@ -56,36 +57,26 @@ $ podman login registry.redhat.io
 ----
 endif::[]
 
-. The `oc adm catalog mirror` command extracts the contents of an index image to generate the manifests required for mirroring. You can choose either of the following:
-+
---
-* Allow the default behavior of the command to automatically mirror all of the image content from the index image, as well as the index image itself, to your mirror registry after generating manifests.
-* Add the `--manifests-only` flag to only generate the manifests required for mirroring, but do not actually mirror the image content to the registry yet. This can be useful for reviewing what will be mirrored, and it allows you to make any changes to the mapping list if you only require a subset of packages. You can then use that file with the `oc image mirror` command to mirror the modified list of images in a later step.
-+
-[NOTE]
-====
-The `--manifests-only` flag is intended for advanced selective mirroring of content from the catalog. The `opm index prune` command, if you used it previously to prune the index image, is suitable for most use cases.
-====
---
-+
-On your workstation with unrestricted network access, run the following command:
+. The `oc adm catalog mirror` command extracts the contents of an index image to generate the manifests required for mirroring. The default behavior of the command generates manifests, then automatically mirrors all of the image content from the index image, as well as the index image itself, to your mirror registry. Alternatively, if your mirror registry is on an airgapped host, you can first mirror the content to removable media, move the media to the disconnected environment, then mirror the content from the media to the registry.
+
+** *Option A:* If your mirror registry is on the same network as your workstation with unrestricted network access, run the following command on the workstation:
 +
 [source,terminal]
 ----
 $ oc adm catalog mirror \
     <index_image> \//<1>
-    <mirror_registry>:<port> \//<2>
+    <mirror_registry>:<port>/<namespace> \//<2>
     [-a ${REG_CREDS}] \//<3>
     [--insecure] \//<4>
     [--index-filter-by-os='<platform>/<arch>'] \//<5>
     [--manifests-only] <6>
 ----
 <1> Specify the index image for the catalog you want to mirror. For example, this might be a pruned index image that you created previously, or one of the source index images for the default catalogs, such as `{index-image-pullspec}`.
-<2> Specify the target registry to mirror the Operator content to.
+<2> Specify the target registry and namespace to mirror the Operator content to, where `<namespace>` is any existing namespace on the registry. For example, you might create an `olm-mirror` namespace to push all mirrored content to.
```

Comment:
If you want users to make a dedicated namespace for the registry before they get here, I'd make it a prereq.